apps/passport-server/src/services/devconnectPretixSyncService.ts

```diff
@@ -533,7 +533,8 @@ export class DevconnectPretixSyncService {
             devconnect_pretix_items_info_id: existingItem.id,
             is_deleted: false,
             is_consumed: false,
-            position_id: id.toString()
+            position_id: id.toString(),
+            secret: order.secret
```

Comment:
❗ it's important that this is the **position** secret, not the order secret. each position ends up being a ticket in our database, not each order.